pubmed_parser/pubmed_web_parser.py

```diff
@@ -138,6 +138,12 @@ def parse_pubmed_web_tree(tree):
         for article_id in article_ids:
             if article_id.attrib.get("idtype") == "doi":
                 doi = article_id.text
+    
+    language = tree.xpath("//language")
+    try:
+        language = language[0].text
+    except:
```

Comment:
Never have a blank "except" statement, because you might hide all kinds of bugs in this way. Only catch specific exceptions. I.e., write `except IndexError` (I think that's the specific error, right?).

Reply:
You are right! That's the error that occurs.